app-backend/api/src/main/scala/token/Token.scala

```diff
@@ -100,7 +100,7 @@ object TokenService extends Config {
 
     val params = FormData(
       "api_type" -> "app",
-      "grant_type" -> "urn:ietf:params:oauth:grant-type:jwt-bearer",
+      "grant_type" -> "refresh_token",
```

Comment:
Have you tested to see if this works if you disable `Legacy: Delegation Refresh Token` in the development Auth0 settings?

Reply:
When I was testing things, the refresh token operations were using the `Web Management API` client in Auth0 vs. the `Web` client. `Legacy: Delegation Refresh Token` is disabled under the `Web` client, but enabled under the `Web Management API` client.

Reply:
I disabled the option in the Web Management API client before testing it just now.

I think there is still more to unpack here. I disabled it and the endpoints still work, but I think we actually need to figure out how to use the `access_token` as opposed to the `id_token` we expect. An `access_token` isn't the same thing as a JWT (it's an opaque token) and we don't have a way to verify it.

The docs seem to suggest:
 >  If the scope of the initial token included openid, then a new id_token will be in the response as well.

And I've verified in https://github.com/raster-foundry/raster-foundry/blob/develop/app-frontend/src/app/services/auth/auth.service.js#L266 that we seem to be actually doing that.